docs/05_community-dev-tools/index.md

```diff
@@ -0,0 +1,50 @@
+
+
+## Overview
+This is a list of developer tools which are built by members of the EOSIO community, curated by the Developer Relations team at Block.one.
+
+If you would like to submit a tool to be added to the list, click **Edit** at the right hand side of this page and submit a Pull Request or fill in [this form](https://blockone.typeform.com/to/gA4F0Q).
```

Comment:
Probably it is better to track the "add a tool request" in one centralized place other than using Github issues. It will clutter the discussion/issues on tutorials/documentation